srcpkgs/wiredtiger/template

```diff
@@ -1,9 +1,9 @@
 # Template file for 'wiredtiger'
 pkgname=wiredtiger
 version=2.9.3
-revision=2
+revision=3
 build_style=gnu-configure
-configure_args="--enable-zlib --enable-lz4 --enable-snappy"
+configure_args="--enable-verbose --enable-zlib --enable-lz4 --enable-snappy"
 makedepends="zlib-devel liblz4-devel snappy-devel"
 short_desc="an high performance, scalable, platform for data management"
```

Comment:
```suggestion
short_desc="High performance, scalable, platform for data management"
```
Check xlint. It would be nice if you would incorporate this change into this pr.